src/app/app.ts

```diff
@@ -27,6 +27,8 @@ creator.on('commandRun', (command, _, ctx) =>
 creator
 	.withServer(
 		new GatewayServer((handler) => client.ws.on(<WSEventType>'INTERACTION_CREATE', handler)),
+		// new FastifyServer(server, {alreadyListening: false}),
```

Comment:
Can this be reverted since there is not change?

Reply:
removed. Should I remove ScoapFastifyServer.ts too as it is not used right now. Could be used for launching scoap squad polls via htttp, not sure if this is of interest still? I set this up in the beginning because the specifications of the feature included invoking the command via a form from the website.

Reply:
Yes please remove it. I think keeping things simple is best even though tons of ideas get thrown around all the time. For the discord APIs we can spin up a separate package and that source code will exist in it's own directory. That way the degen commands are organized nicely

Reply:
ok done